packages/block-library/src/image/image.js

```diff
@@ -109,7 +109,7 @@ export default function Image( {
 	const [ isEditingImage, setIsEditingImage ] = useState( false );
 	const [ externalBlob, setExternalBlob ] = useState();
 	const clientWidth = useClientWidth( containerRef, [ align ] );
-	const isResizable = ! isWideAligned && isLargeViewport;
+	const isResizable = false;
```

Comment:
The resize option messes with fixed grid layout as it attached `style=` height and width attribs, so we would need to add an additional prop to the Image block to make this toggleable